src/core/ledger/ledger.test.js

```diff
@@ -1598,6 +1598,154 @@ describe("core/ledger/ledger", () => {
     });
   });
 
+  describe("Grain Integrations", () => {
+    const getEmptyDistribution = (time: number = 1) => ({
+      credTimestamp: time,
+      allocations: [],
+      id: uuid.random(),
+    });
+    describe("Toggle Integrations", () => {
+      it("can enable integrations", () => {
+        const ledger = new Ledger();
+        ledger.enableIntegrationTracking();
+        expect(ledger.eventLog()).toEqual([
+          {
+            "action": {
+              type: "TOGGLE_INTEGRATION",
+            },
+            ledgerTimestamp: 4,
```

Comment:
??? is this relying on a stable test order to output stable auto-incremented timestamps? Seems flimsy. Would be better to reset the clock in a "beforeEach" or something.